PolishWithBlondes/src/components/home-info/home-info.scss

```diff
@@ -0,0 +1,42 @@
+.info-outer-container {
+  background-color: var(--secondary-background);
+  padding: 70px 0px;
+}
+
+.info-container {
+  max-width: 1060px;
```

Comment:
1060px is your max container width, right?
It will probably be used in different layouts as well. Maybe we could make a variable for it